presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergQueryRunner.java

```diff
@@ -176,14 +177,9 @@ public static void main(String[] args)
     {
         Logging.initialize();
         Optional<Path> dataDirectory = Optional.empty();
-        if (args.length > 0) {
-            if (args.length != 1) {
-                log.error("usage: IcebergQueryRunner [dataDirectory]\n");
-                log.error("       [dataDirectory] is a local directory under which you want the iceberg_data directory to be created.]\n");
-                System.exit(1);
-            }
-
-            File dataDirectoryFile = new File(args[0]);
+        Optional<String> suppliedDataDirectoryPath = getProperty("DATA_DIR");
```

Comment:
getProperty("DATA_DIR") would return Optional.empty() if the property is not set, so you don't need to use another variable suppliedDataDirectoryPath

Reply:
I had introduced the variable so that we do not need to call getProperty twice, first on L181 and then on L182.